setup.py

```diff
@@ -0,0 +1,68 @@
+# coding: utf-8
+
+import os
+import re
+import setuptools
+
+
+def ascii_bytes_from(path, *paths):
+    """
+    Return the ASCII characters in the file specified by *path* and *paths*.
+    The file path is determined by concatenating *path* and any members of
+    *paths* with a directory separator in between.
+    """
+    file_path = os.path.join(path, *paths)
+    with open(file_path) as f:
+        ascii_bytes = f.read()
+    return ascii_bytes
+
+
+# read required text from files
+thisdir = os.path.dirname(__file__)
+init_py = ascii_bytes_from(thisdir, "src", "histolab", "__init__.py")
+readme = ascii_bytes_from(thisdir, "README.md")
+# This allows users to check installed version with:
+# `python -c 'from histolab import __version__; print(__version__)'`
+version = re.search('__version__ = "([^"]+)"', init_py).group(1)
+
+install_requires = [
+    "matplotlib==3.2.1",
+    "numpy==1.18.4",
+    "openslide-python==1.1.1",
+    "Pillow==7.1.2",
+    "scikit-image==0.16.2",
+    "scipy==1.4.1",
+]
+
+test_requires = ["pytest", "coverage", "pytest-cov", "coveralls", "pytest-mpl"]
+
+setuptools.setup(
+    name="histolab",
+    version=version,
+    maintainer="Histolab Developers",
+    mantainer_email="ernesto.arbitrio@gmail.com",
```

Comment:
typo:
```suggestion
    maintainer_email="ernesto.arbitrio@gmail.com",
```

Reply:
yes it's a typo and BTW i don't wanna use my email, do we wanna create a project discussion list ? eg. histolab-discussions@python.org ? XD

Reply:
Oh man, you're thinking so big :D 

I think this is a bit overkill for the moment... @nicolebussola what do you think?

Reply:
as per discussion, but let leave my email for the moment